system/mp/mpcomm.cpp

```diff
@@ -1987,10 +1987,25 @@ CMPConnectThread::CMPConnectThread(CMPServer *_parent, unsigned port, bool _list
             if (!mpSoMaxConn)
                 mpSoMaxConn = env->getPropInt("EnvSettings/ports/mpSoMaxConn", 0);
         }
-        parent->mpTraceLevel = env->getPropInt("EnvSettings/mpTraceLevel", 0);
+        unsigned mpTraceLevel = env->getPropInt("EnvSettings/mpTraceLevel", 0);
+        switch (mpTraceLevel)
+        {
+            case 0:
+                parent->mpTraceLevel = InfoMsgThreshold;
+                break;
+            case 1:
+                parent->mpTraceLevel = DebugMsgThreshold;
+                break;
+            case 2:
+                parent->mpTraceLevel = ExtraneousMsgThreshold;
+                break;
+            default:
+                parent->mpTraceLevel = ExtraneousMsgThreshold + 10;
+                break;
+        }
     }
 #else
-    parent->mpTraceLevel = getComponentConfigSP()->getPropInt("logging/@detail", 0);
+    parent->mpTraceLevel = getComponentConfigSP()->getPropInt("logging/@detail", InfoMsgThreshold);
```

Comment:
as most components have detail: 100, the new default here is not going to be used.
hence mpTraceLevel will normally be 100